app/src/main/java/io/homeassistant/companion/android/qs/TileExtensions.kt

```diff
@@ -66,7 +66,7 @@ abstract class TileExtensions : TileService() {
         fun setTileData(context: Context, tileId: String, tile: Tile, integrationUseCase: IntegrationRepository): Boolean {
             Log.d(TAG, "Attempting to set tile data for tile ID: $tileId")
             val tileDao = AppDatabase.getInstance(context).tileDao()
-            val tileData = tileDao.get(tileId)
+            val tileData = tileDao.getSync(tileId)

```

Comment:
This whole function should become a suspend function but since all you are actually doing is a rename it's fine.

Reply:
Updated in https://github.com/home-assistant/android/pull/2541